wordpress-mobile/WordPress-iOS
# Fix Scheduled Post displayed status

Fixes #13655
Fixes #13654

- Addresses a few bugs with post status not being changed when scheduling a post.
- Adds a confirmation prompt when a post is Published (seems like the logic for this was not checking the original post status)

To test:
- Schedule or update a post and check the resulting labels

PR submission checklist:
